Esmorga-Backend/esmorga-backend
# [MOB-18] BE - Project Basic Configs

## Pull Request Checklist

- [x] **[MOB US' Link](https://mobilemakers.atlassian.net/browse/MOB-18)**
- [x] **Add PR url to US**
- [x] **Add variables to doc**

## Testing Guide

1. Create the .env file at the project root.
2. Run the project and check that everything is fine.
```
npm run start:dev
```
⚠️ Right now these are the only variables in the .env file:
```
MONGODB_URI=mongodb://localhost/nest
REDIS_HOST=localhost
REDIS_PORT=6379
APP_PORT=3000
```
